docs/Home/index.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
-import { Code } from '@deque/cauldron-react';
+import { Code, Link } from '@deque/cauldron-react';
 
 const Home = () => (
   <div>
-    <h1>Cauldron React</h1>
+    <h1>Cauldron React by Deque Systems</h1>
+    <h2>Purpose</h2>
+    <p>
+      These packages include everything from typography and colors to components
+      like custom form controls. The design and interactions are intended to
+      show how Deque would solve these problems. It's how we approach them in
+      our prouducts.
```

Comment:
```suggestion
      These packages include everything from typography and colors to components, like custom form controls. Considerations in design and interactions show how Deque would solve these problems in our products.
```

Also expand on 'these problems' -- what problems? :)